eslzArm/eslz-portal.json

```diff
@@ -108,7 +108,37 @@
 								}
 							],
 							"visible": "[equals(steps('lzSettings').subOrgsOption, 'Single')]"
-            			}
+            			},
+                        {
+                            "name": "cuaSettingsInfo",
+                            "type": "Microsoft.Common.InfoBox",
+                            "visible": true,
+                            "options": {
+                                "text": "See Enterprise Scale Repository Docs > Wiki > Deploying-Enterprise-Scale-CustomerUsage for complete info. Microsoft can identify the deployments of the Azure Resource Manager templates with the deployed Azure resources. The usage of this data is governed by Microsoft Trust and Privacy policies. The customer usage identifier is used for telemetry purposes.",
+                                "uri": "https://github.com/Azure/Enterprise-Scale/blob/main/docs/EnterpriseScale-CustomerUsage.md",
```

Comment:
Is this URL correct? Should point to Wiki, no?

Reply:
corrected URL